src/currencies/sortByMarketcap.ts

```diff
@@ -107,3 +107,36 @@ export const useCurrenciesByMarketcap = <C extends Currency>(
   const tickers = useMarketcapTickers();
   return tickers ? sortByMarketcap(currencies, tickers) : currencies;
 };
+
+type CurrenciesByMarketcapResult = {
+  loading: boolean;
+  error?: Error;
+  currencies?: any[];
+};
+
+export const useCurrenciesByMarketcapWithStatus = <C extends Currency>(
```

Comment:
@ThomasLaforge can you try and add a unit test case for this hook. should be pretty straight forward creating one starting from the other hook in the file. :pray:

Reply:
There is no tests for the equivalent hook. Only for function in it.
I can check how to test a hook ^^